types/activex-excel/index.d.ts

```diff
@@ -3156,10 +3174,84 @@ declare namespace Excel {
         readonly _Default: string;
         _Evaluate(Name: any): any;
         _FindFile(): void;
-        _MacroOptions(Macro?: any, Description?: any, HasMenu?: any, MenuText?: any, HasShortcutKey?: any, ShortcutKey?: any, Category?: any, StatusBar?: any, HelpContextID?: any, HelpFile?: any): void;
-        _Run2(Macro?: any, Arg1?: any, Arg2?: any, Arg3?: any, Arg4?: any, Arg5?: any, Arg6?: any, Arg7?: any, Arg8?: any, Arg9?: any, Arg10?: any, Arg11?: any, Arg12?: any, Arg13?: any, Arg14?: any, Arg15?: any, Arg16?: any, Arg17?: any, Arg18?: any, Arg19?: any, Arg20?: any, Arg21?: any, Arg22?: any, Arg23?: any, Arg24?: any, Arg25?: any, Arg26?: any, Arg27?: any, Arg28?: any, Arg29?: any, Arg30?: any): any;
+        _MacroOptions(
+            Macro?: any,
+            Description?: any,
+            HasMenu?: any,
+            MenuText?: any,
+            HasShortcutKey?: any,
+            ShortcutKey?: any,
+            Category?: any,
+            StatusBar?: any,
+            HelpContextID?: any,
+            HelpFile?: any
```

Comment:
Perhaps if all the function arguments are `any` the arguments should be on a single line, or however many lines it takes; instead of one argument per line?

Reply:
This is also prettier's formatting system; if the function declaration, all the way to the `{`, does not fit within `printWidth`, all the arguments are printed one per line.